napari/utils/events/containers/_selection.py

```diff
@@ -124,21 +139,21 @@ def clear(self, keep_current: bool = False) -> None:
             self._current = None
         super().clear()
 
-    def toggle(self, obj: _T):
+    def toggle(self, obj: _T) -> None:
         """Toggle selection state of obj."""
         self.symmetric_difference_update({obj})
 
-    def select_only(self, obj: _T):
+    def select_only(self, obj: _T) -> None:
         """Unselect everything but `obj`. Add to selection if not present."""
         self.intersection_update({obj})
         self.add(obj)
 
     @classmethod
-    def __get_validators__(cls):
+    def __get_validators__(cls) -> Generator:
         yield cls.validate
 
     @classmethod
-    def validate(cls, v, field: 'ModelField'):
+    def validate(cls, v: Union["Selection", Dict], field: 'ModelField') -> "Selection":  # type: ignore[override]
```

Comment:
```suggestion
    def validate(cls, v: Union['Selection', Dict], field: 'ModelField') -> 'Selection':  # type: ignore[override]
```

Reply:
Does the type of quotes really matter here? And if it does, wouldn't it be better to enforce it with an automated pre-commit check?